src/CMakeLists.txt

```diff
@@ -79,12 +81,12 @@ if(CMAKE_INTERPROCEDURAL_OPTIMIZATION)
     # Policy that enables INTERPROCEDURAL_OPTIMIZATION
     set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)
     cmake_policy(SET CMP0069 NEW)
-    set(CMAKE_INTERPROCEDURAL_OPTIMIZATION ON)
     if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
       # Clang has some issues with thinLTO (segfaults on final linking)
       # This enables full LTO through CMake
       # FullLTO is slow, but may yield performance improvements.
-      add_compile_options(-flto=full)
+      set(CMAKE_CXX_COMPILE_OPTIONS_IPO "-flto")
+      #add_compile_options(-flto=full)
```

Comment:
Why is this commented out?

Reply:
There is not a standard way to force FAT LTO building. They are both hacky. The add_compile_option adds it after the thinLTO so it override replacement, where the CMAKE_CXX_COMPILE_OPTIONS_IPO is an undocumented variable to replace it. I am not sure which is better. Thoughts? I guess the original way might be better since it might preserve other OPTIONS_IPO flags that may eventually get set by CMake if they get added to clang in the future.